src/Razor/src/Microsoft.VisualStudio.RazorExtension/ServiceHubCore/Microsoft.VisualStudio.Razor.TagHelperProviderCore64.AssemblySearchPathsConfig.json

```diff
@@ -0,0 +1,3 @@
+﻿[
+  "..\\..\\managedLanguages\\VBCSharp\\LanguageServices"
```

Comment:
I sent you the wrong path yesterday, this should be "..\\..\\managedLanguages\\VBCSharp\\LanguageServices\\ServiceHubCore"